README.md

```diff
@@ -1,2 +1,49 @@
-# akka-sample-crud-jpa
-Sample of CRUD application with JPA and Akka
+# Sample of CRUD application with JPA and Akka
+
+## How to run
+
+```
+mvn compile exec:java
```

Comment:
I've tried this. I'm facing 2 issues:

* once the message wrt the server is ready I try a couple of the `curl` commands listed below and both failed with the message: `curl: (7) Failed to connect to localhost port 8080: Connection refused`
* back on the Maven terminal I see `Press RETURN to stop...` but pressing `RETURN` doesn't stop the process. Note: I'm testing on a terminal inside IntelliJ (terminal plugin) and this could be a side-effect of such combination.

Reply:
yes, the reading from stdin should be removed, we already did that in the http scala sample but I started from the java sample, https://github.com/akka/akka-http-quickstart-java.g8/issues/15

Reply:
It works for me `mvn compile exec:java`, but don't press ENTER in that terminal